### DISCUSSION TITLE
OpenMediaVault unable to see disks in a container

### DISCUSSION THREAD

**Discussion by dbzx6r** [tteck/Proxmox#1263] [Help Needed]
I was unable to get OpenMediaVault to see any of the disks no matter what I did after using the script and using it in a proxmox container. I mounted the drives in Proxmox, attempted to mount the drives in OMV, no matter what I did the drives would not mount and be viewable.

However, using the OMV ISO, I was able to instantly get the disks to be seen and was able to get it set up.

Does OpenMediaVault not work in a container? Or was there something I was missing while using the script?

**Comment by MrFiestaxxl** [comment]
I have the same issue. I tried to use a mountpoint as share but the omv plugin that allows to mount a folder as a share is useless at this point. So most likely i have to use a vm instead and mount a disk into it.

**Comment by OsipovOV** [comment]
I have the same issue

**Comment by Porencephaly** [comment]
+1

**Comment by andreyxc** [comment]
+1

**Comment by wokawoka** [comment]
+1

**Comment by ttoopp** [comment]
Hi, is there a instruction, how to add a Storage to the OMV Script installation?

**Comment by Jens-Wymeersch** [comment]
Same here. Anyone sorted this out ?

**Comment by ScienceWolf666** [comment]
> I was unable to get OpenMediaVault to see any of the disks no matter what I did after using the script and using it in a proxmox container. I mounted the drives in Proxmox, attempted to mount the drives in OMV, no matter what I did the drives would not mount and be viewable.
> 
> However, using the OMV ISO, I was able to instantly get the disks to be seen and was able to get it set up.
> 
> Does OpenMediaVault not work in a container? Or was there something I was missing while using the script?

+1 same problem

**Comment by gitandriu** [comment]
Same

**Comment by edrzns** [comment]
+1
Exactly same issue here, no matter what I've tried, either bind mounting or adding drives as a block devices to OMC CT, OMV refuses to see them and throws error while you try to refresh and look for disks.

If anyone found some solution would appreciate

**Comment by iodize6399** [comment]
I tried using `lxc.mount.entry` and bind mounts, but neither works with omv. Oddly, I can see the mounts and `cat` files from the CT's shell. However, I get an error 500 when trying to change settings via the GUI. Right now, the container is pretty much useless.

![image](https://github.com/tteck/Proxmox/assets/4197707/79ed81bc-53dc-467e-9b29-76df49bb8b19)

**Comment by lcontursi** [comment]
+1 same problem

**Comment by fl3usner** [comment]
https://forum.openmediavault.org/index.php?thread/45664-no-disk-device-found-to-share/&postID=333286#post333286

It says that currently it's not working via container. Workaround: Install in VM
